addons/block_code/ui/blocks/utilities/parameter_input/parameter_input.gd

```diff
@@ -1,11 +1,15 @@
 @tool
 extends MarginContainer
 
+signal drag_started
+
 const OptionData = preload("res://addons/block_code/code_generation/option_data.gd")
 const Types = preload("res://addons/block_code/types/types.gd")
 
 signal modified
 
+const DRAG_MINIMUM_THRESHOLD = 25
```

Comment:
Nit: we have a script with all the UI constants at `addons/block_code/ui/constants.gd`.